.github/dependabot.yml

```diff
@@ -11,30 +11,18 @@ updates:
     schedule:
```

Comment:
I'm not so familiar with this -- could someone explain the changes on this file for me?